calculate.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <string>
+
+std::string calculate(const std::string& command, int first, int second) {
+    int res{};
+    if(command == "add"){
+       res = first + second;
+    }else if(command == "subtract"){
+         res = first - second;
+    }else if(command == "multiply"){
+         res = first * second;
+    }else if(command == "divide"){
+          if(second != 0){
+           res = first/second;
+          } else{ return "Division by 0";}
+          
+        } else{return "Invalid data";}
```

Comment:
It't the best if you use some auto formatter. Your IDE should handle indentations for you.
Here your code is indented for 4, 7, 9, 10 or 11 spaces. It is not consistent. 
Always use the same amount of spaces for one indentation (usually 4).